python-lib/gluonts_forecasts/model_handler.py

```diff
@@ -223,3 +223,11 @@ def get_model_label(model_name):
         return None
     else:
         return model_descriptor.get(LABEL, "")
+
+
+def get_model_name_from_label(model_label):
```

Comment:
Perhaps there is a simpler way to do this?

Reply:
sure, I will do a one-liner :)

Reply:
done in d42615b

Reply:
Nice!